src/components/common/FileUploadPreview.vue

```diff
@@ -66,7 +72,7 @@ export default class FileUploadPreview extends Vue {
     if (this.inputFile) {
       this.fileUpload = this.inputFile
       this.$nextTick(() => {
```

Comment:
I'd prefer to see `await this.$nextTick()` to make this all async. Otherwise, `this.isFileValid()` will be executed in the tick after this method (ie, after `this.emitFileSelected()`), which may be incorrect.

Reply:
updated to use `await this.$nextTick()`